### PR TITLE
[zh-cn] Add kubectl_create_secret.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret.md

### CODE REVIEW

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret.md [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+
+## {{% heading "seealso" %}}
+
+<!--
+* [kubectl create](../)	 - Create a resource from a file or from stdin
+* [kubectl create secret docker-registry](../kubectl_create_secret_docker-registry/)	 - Create a secret for use with a Docker registry
+* [kubectl create secret generic](../kubectl_create_secret_generic/)	 - Create a secret from a local file, directory, or literal value
+* [kubectl create secret tls](../kubectl_create_secret_tls/)	 - Create a TLS secret
+-->
+* [kubectl create](../) - 从文件或标准输入创建资源
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret.md:514]:
```suggestion
* [kubectl create](../) - 基于文件或标准输入创建资源
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret.md [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+* [kubectl create secret docker-registry](../kubectl_create_secret_docker-registry/) - 创建用于 Docker 仓库类型 Secret
+* [kubectl create secret generic](../kubectl_create_secret_generic/) - 从本地文件、目录或文字值创建 Secret
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret.md:516]:
```suggestion
* [kubectl create secret generic](../kubectl_create_secret_generic/) - 基于本地文件、目录或文字值创建 Secret
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret.md [hunk 3]
```diff
@@ -0,0 +1,517 @@
...
+## {{% heading "seealso" %}}
+
+<!--
+* [kubectl create](../)	 - Create a resource from a file or from stdin
+* [kubectl create secret docker-registry](../kubectl_create_secret_docker-registry/)	 - Create a secret for use with a Docker registry
+* [kubectl create secret generic](../kubectl_create_secret_generic/)	 - Create a secret from a local file, directory, or literal value
+* [kubectl create secret tls](../kubectl_create_secret_tls/)	 - Create a TLS secret
+-->
+* [kubectl create](../) - 从文件或标准输入创建资源
+* [kubectl create secret docker-registry](../kubectl_create_secret_docker-registry/) - 创建用于 Docker 仓库类型 Secret
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret.md:515]:
```suggestion
* [kubectl create secret docker-registry](../kubectl_create_secret_docker-registry/) - 创建 Docker 仓库所用的 Secret
```

**Review by windsonsea — COMMENTED** [review]
nits
/approve

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret.md [hunk 4]
```diff
@@ -0,0 +1,517 @@
...
+
+ A generic type secret indicate an Opaque secret type.
+
+ A tls type secret holds TLS certificate and its associated key.
+-->
+创建指定类型的 Secret：
+
+- docker-registry 类型 Secret 用于访问容器镜像仓库。
+
+- 通用类型 Secret 表示不透明 Secret 类型。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_create/kubectl_create_secret.md:28]:
```suggestion
- generic 类型 Secret 表示不透明 Secret 类型。
```

**Pull request merged**